wiki/Tournaments/CWC/2020/en.md

```diff
@@ -0,0 +1,190 @@
+
+The **osu!catch World Cup 2020** (***CWC 2020***) is a country-based osu!catch tournament hosted by the [osu! team](/wiki/People/The_Team). It is the 9th installment of the osu!catch World Cup.
+
+## Tournament Schedule
+
+| Event | Timestamp |
+| --: | :-- |
+| Registration Phase | 2020-04-07/2020-04-21 |
+| Live Drawings | 2020-05-02 (14:00 UTC) |
+| Group Stage | 2020-05-09/2020-05-10 |
+| Round of 16 | 2020-05-16/2020-05-17 |
+| Quarterfinals | 2020-05-23/2020-05-24 |
+| Semifinals | 2020-05-30/2020-05-31 |
+| Finals | 2020-06-06/2020-06-07 |
+| Grand Finals | 2020-06-13/2020-06-14 |
+
+## Prizes
+
+| Placing | Prize(s) |
+| :-: | :--- |
+| ![Gold Crown](/wiki/shared/crown-gold.png "1st place") | $150 per team member, unique profile badge, "osu!catch Champion" user title for one year |
+| ![Silver Crown](/wiki/shared/crown-silver.png "2nd place") | $80 per team member, Unique profile badge |
+| ![Bronze Crown](/wiki/shared/crown-bronze.png "3rd place") | $40 per team member, Unique profile badge |
+
+![CWC 2019 Winner Badge](img/badge.png)
+
+## Organisation
+
+The osu!catch World Cup 2019 was run by various community members by distributing the multitude of tasks into various fields of responsibility.
+
+| Position | Members |
+| :-- | :-- |
+| Manager | ![][flag_AR] [juankristal](https://osu.ppy.sh/users/443656), ![][flag_CL] [WalterToro](https://osu.ppy.sh/users/5281416) |
+| Map Selector | *to be announced* |
+| Commentator | *to be announced* |
+| Referee | ![][flag_PL] [Benzopirene](https://osu.ppy.sh/users/1887068), ![][flag_ES] [Deif](https://osu.ppy.sh/users/318565), ![][flag_CH] [Icerite](https://osu.ppy.sh/users/7226287), ![][flag_AU] [ill onion](https://osu.ppy.sh/users/8306102) ![][flag_DE] [p3n](https://osu.ppy.sh/users/123703), ![][flag_US] [tigereyes144](https://osu.ppy.sh/users/6499811), ![][flag_CL] [WalterToro](https://osu.ppy.sh/users/5281416), ![][flag_GB] [Yazzehh](https://osu.ppy.sh/users/7068973) |
+| Statistician | ![][flag_DE] [Nwolf](https://osu.ppy.sh/users/1910766) |
+
+## Links
+
+- [Discussion thread](https://osu.ppy.sh/community/forums/topics/1046500)
+- [Livestream](https://www.twitch.tv/osulive)
+- **[Click here to sign-up!](https://osu.ppy.sh/community/tournaments/24)**
+
+---
+
+## Ruleset
+
+### Tournament Rules
+
+1. The osu!catch World Cup is a country-based team tournament, played on the osu!catch game mode.
+   - While this competition is planned as a 3 versus 3 setup, this might change depending on the amount of incoming registrations.
+2. Map scoring will use Score V2.
+3. The maps for each round will be announced by the mapset selectors in advance on the Sunday before the actual matches take place. Only these will be used during the respective matches.
+   - There will be a [Hidden](/wiki/Game_Modifiers), [HardRock](/wiki/Game_Modifiers) and a [DoubleTime](/wiki/Game_Modifiers) bracket.
+   - One map will be given as a tiebreaker map. This map will only be played in case of a tie.
```

Comment:
This has probably been this way since forever, but it's not clear if this means a "tie on one map" or a "tie at match point". Should probably be clarified if it isn't already elsewhere.

Point 8. references a beatmap ending in a "draw", so perhaps this the intended distinction?

Also applies under "Match Instructions" if a change / clarification is made here.